_rules/document-has-headings-for-main-section-of-content-047fe0.md

```diff
@@ -0,0 +1,444 @@
+---
+id: 047fe0
+name: Document has heading for main section of content
+rule_type: atomic
+description: |
+  This rule checks that the main section of content starts with a heading
+accessibility_requirements:
+  wcag-technique:H69: # Providing heading elements at the beginning of each section of content
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility Tree
+  - CSS Styling
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Jean-Yves Moyen
+  assets:
+    - _The Three Kingdoms_ by Luo Guanzhong, translation by Yu Sumei (Tuttle publishing, 2014, ISBN 9780804843935)
+    - Image from a Ming Dynasty edition of the Romance of the Three Kingdoms, original kept in the library holdings of Peking University.
+---
+
+## Applicability
+
+This rule applies to any [HTML web page][].
+
+## Expectations
+
+The first non-[decorative][] node in the [flat tree][] which is inside the [main block of content][] of the test target and has a non-empty [accessible name][]:
```

Comment:
This needs to be phrased in terms of "each test target". Also, use the conditional list phrase "all the following is true"